### DISCUSSION TITLE
USB Charge Only

### DISCUSSION THREAD

**Discussion by Cele931** [Mr-MIBonk/M.I.B._More-Incredible-Bash#280] [Q&A]
Hello I want to activate Android Auto and apple car pla on my Audi A3 8V e-tron. A already activated it with the MIB tool. But now I saw that I only a USB port with charge only... Is there any instruction to replace the USB port with an full function Port?

I can not logging in on the mibwiki.one via discord.

I hope someone can help me

Thanks a lot.

Best regards

**Comment by harman-f** [comment]
you have to join the MIB Discord channel 1st.
